src/App.js

```diff
@@ -6,30 +6,67 @@ import { useState } from "react";
 import TodoCards from "./components/TodoCards/TodoCards";
 import AddForm from "./components/AddForm/AddForm";
 import AddButton from "./components/AddButton/AddButton";
+import Popup from "./components/Popup/Popup";
 
 function App() {
   const [todoItems, setTodoItems] = useState(todos);
-  const [activeClass, setActiveClass] = useState(true);
+  const [activeClass, setActiveClass] = useState({
+    form: false,
+    popup: false,
+    addbtn: true,
+  });
+  const [postId, setPostId] = useState(null);
 
   const addTodo = (todo) => {
+    if (!todo.text.trim()) return;
     todo.id = todoItems.length + 1;
     const tmpArr = [...todoItems, todo];
     setTodoItems(tmpArr);
   };
 
-  const toggleActivity = () => {
-    setActiveClass(!activeClass);
+  const toggleActivity = (obj) => {
+    let tmpObj = { ...activeClass, ...obj };
+    setActiveClass(tmpObj);
+  };
+
+  const getId = (id) => setPostId(id);
```

Comment:
what is going on here? it's called `getId` but calls `setPostId`.
Is it set or get?